styles.css

```diff
@@ -0,0 +1,6 @@
+img {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
```

Comment:
i suggest adding an image to to your index files src to fix the broken image link/path